spec/features/admin_can_view_all_owners_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "rails_helper"
+
+RSpec.feature "Admin is logged in" do
+  scenario "goes to view all owners" do
+# As a logged-in admin,
+# when I am on my dashboard,
+# and I click on "view all owners,"
+# I can see a page that lists all the owners,
+# and for each owner,
+# I can see a link to that owner's show page,
+# see a link to that owner's dashboard,
+# see a link to edit that owner,
+# see a link to suspend that owner,
+# and see a link to delete that owner.
+    admin_user = create(:user)
```

Comment:
I like that you scoped the expectations into classes, should be doing that more and refactor some old tests to do that once are styling is in place. I would like to see the user story comments deleted before merging into master.